rfcs/20210403-DMF-v1.0.0.md

```diff
@@ -0,0 +1,122 @@
+
+# nanoMFG Software Planning Document
+<!-- Replace text below with long title of project:short-name -->
+## nanoMFG Data Management Framework
+### Target Release: #.#.# : Month Day, 20##
+
+## Project Team
+<!-- Complete table for all team members 
+ roles: lead, developer, reviewer, contributer
+ status: active, inactive
+-->
+Name | Role | github user | nanohub user | email | status
+---|---|---|---|---|---
+Joe Shmo | lead | jshmo | jshmo | jshmo@nanocool.co | active
+Alice Land | developer | alice1 | alice2 | alice@wonderland | inactive
+
+**nanoMFG Github Team(s):** @coolteam
+**nanoHUB Group(s):**
+
+## 1. Introduction
+
```

Comment:
maybe something should go here with a very brief statement of the tool and a very general description of what it's for and who the users are intended to be?  e.g., This is cooltool, which simulates the process of manufacturing ... and should be useful for experimentalists who want to simulate this process many times to select good parameters before they actually run more costly physical experiments

Reply:
Or maybe this is what you have as 1.1, but I think could be 1.0...